beekeeper-metadata-cleanup/src/main/java/com/expediagroup/beekeeper/metadata/cleanup/context/CommonBeans.java

```diff
@@ -81,11 +82,11 @@ public CloseableMetaStoreClientFactory metaStoreClientFactory() {
     return new HiveMetaStoreClientSupplier(metaStoreClientFactory, hiveConf, name);
   }
 
-  @Bean
-  public HiveClient hiveClient(
+  @Bean(name = "hiveClientFactory")
```

Comment:
why name it?

Reply:
Keeping it consistent with the other beans that have seemingly generic methods, but return a specific hive implementation